fusor-ember-cli/app/templates/components/review-link.hbs

```diff
@@ -5,20 +5,27 @@
 
   <div class='col-sm-6 col-md-8 col-lg-9'>
     {{#if isExternalURL}}
-      <a href={{value}} target="_blank">{{value}}</a>
+        <a href={{value}} target="_blank">{{value}}</a>
     {{else if isNotALink}}
-      {{value}}
+        {{#if useYieldInstead}}
+          {{yield}}
+        {{else}}
+          {{valueFormatted}}
+        {{/if}}
+        {{#if isPassword}}
+            <i {{action 'showPassword'}} class="fa {{eyeIcon}}"></i>
```

Comment:
these edits for for showing the Director Password as ****** with eye-open/close widget in the Review without a link since it is created by the server.